test/python/transpiler/test_transpile.py

```diff
@@ -220,3 +221,23 @@ def test_already_mapped_via_layout(self):
         cx_qubits_physical = [[ctrl[1], tgt[1]] for [ctrl, tgt] in cx_qubits]
         self.assertEqual(sorted(cx_qubits_physical),
                          [[9, 4], [9, 4]])
+
+    def test_parameterized_circuit_for_simulator(self):
+        """Verify that a parameterized circuit can be transpiled for a simulator backend."""
+        qr = QuantumRegister(2, name='qr')
+        qc = QuantumCircuit(qr)
+
+        theta = sympy.Symbol('theta')
+        qc.rx(theta, qr[0])
+
+        transpile(qc, backend=BasicAer.get_backend('qasm_simulator'))
+
+    def test_parameterized_circuit_for_device(self):
+        """Verify that a parameterized circuit can be transpiled for a device backend."""
+        qr = QuantumRegister(2, name='qr')
+        qc = QuantumCircuit(qr)
+
+        theta = sympy.Symbol('theta')
+        qc.rx(theta, qr[0])
+
+        transpile(qc, backend=FakeMelbourne())
```

Comment:
In these 2 tests can you `assert` on what the expected output should be?

Reply:
:+1: Added in 738353078ee7111a19933672d8e3984623dbc637.